src/middleware.d.ts

```diff
@@ -1,71 +0,0 @@
-/// <reference path="./middleware.js" />
```

Comment:
This doesn't look correct, as there is no point to import JS file which is not going to have any types in it, it only introduces problems for TS compiler when project is configured to ignore JS files.